src/components/accessories/discharge/PatientDischarge.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { AdmissionDTO } from "../../../generated";
+import { initialFields } from "./consts";
+import DischargeForm from "./dischargeForm/DischargeForm";
+import "./styles.scss";
+
+export const PatientDischarge: React.FC = () => {
+  const { t } = useTranslation();
+  const infoBoxRef = useRef<HTMLDivElement>(null);
+  const [shouldResetForm, setShouldResetForm] = useState(false);
+  const [shouldUpdateTable, setShouldUpdateTable] = useState(false);
```

Comment:
As before

Reply:
@axelpezzo I think, we could make simple, by closing the this PR #171 and the PR #173, so you can simply review the P #174

Reply:
Because I think All those works are used and corrected on PR #174

Reply:
please what do you think?